set.mm

```diff
@@ -518224,6 +518263,512 @@ such that every nonzero prime ideal contains a prime element (this is a
+      CUCLZKZWICUDLZMZHUEZIUEZCUFLZUGZWIKZWNWIKZWOWIKZUHZUPZIWLUAHWLUAZWICUILKZ
+      FACUJKZWHWLUKWKACFULZABWLAWLCUQLZCJBWLNZXGNZFGUMZUNWLCWJWHDEXHWJNUOURAWLW
+      IACUSLZWLKZXKWIKZUTWLWIMAXEXLXFWLCXKXHXKNZVAVBAXMBXKCVCLZOZAXPBCVHLZKZAXG
+      BCXQJXIXQNZFGVDAXPPWGXPXKXQKZXRAWGXPFQAXPVEAXTXPAXEXTXFCXQXKXSXNVIVBQXOCX
+      QXKBXSXONZVFVGVSAWLXOCDBXKXHEYAXFXJRVJXKWLWIVKURVLAXBHIWLWLAWNWLKZWOWLKZX
+      BAYBPZYCPZWRXAYEWRPZBWNXOOZBWOXOOZWSWTYFWSYGAWSYGVMYBYCWRAWLXOCDBWNXHEYAX
+      FXJRSTYFWTYHYFWLXOCDBWOXHEYAYEXEWRAXEYBYCXFVNZQYEBWLKZWRAYJYBYCXJVNZQRTYF
+      WLXOXGBCWPJWNWOXHXIYAWPNZAWGYBYCWRFSABXGKYBYCWRGSAYBYCWRVOYDYCWRVPYEWRBWQ
+      XOOYEWLXOCDBWQXHEYAYIYKRVQVRVTWAWBWCWGXDWKWMXCWDHIWLWICWPXHYLWETWF $.
+  $}
+
+  ${
+    $d B x y $.  $d K x y $.  $d R x y $.  $d X x y $.  $d ph x y $.
+    rsprprmprmidlb.0 $e |- .0. = ( 0g ` R ) $.
+    rsprprmprmidlb.b $e |- B = ( Base ` R ) $.
+    rsprprmprmidlb.p $e |- P = ( RPrime ` R ) $.
+    rsprprmprmidlb.k $e |- K = ( RSpan ` R ) $.
+    rsprprmprmidlb.r $e |- ( ph -> R e. IDomn ) $.
+    rsprprmprmidlb.x $e |- ( ph -> X e. B ) $.
+    rsprprmprmidlb.1 $e |- ( ph -> X =/= .0. ) $.
+    $( In an integral domain, an ideal is generated by a prime elements is a
+       prime iff that elememt is prime.  (Contributed by Thierry Arnoux,
+       18-May-2025.) $)
+    rsprprmprmidlb $p |- ( ph -> ( X e. P <-> ( K ` { X } ) e. ( PrmIdeal ` R )
+       ) ) $=
+      ( vx wcel cfv wa adantr eqid vy csn cprmidl ccrg idomcringd crpm wceq a1i
+      eleq2d biimpa rsprprmprmidl cidom cui cun cdif cv cmulr co cdsr wbr wo wi
+      wral wn unitpidl1 biimpar wne idomringd prmidlnr sylancom neneqd pm2.65da
+      crg nelsn syl nelun ax-mp sylanbrc eldifd ad3antrrr ad4antr ellpi simp-4r
+      wb simpllr simplr ad2antrr prmidlc syl23anc orim12da ex anasss ralrimivva
+      isrprm syl12anc eleqtrrdi impbida ) AFCPZFUBEQZDUCQPZAWRRFDEKADUDPZWRADLU
+      EZSAWRFDUFQZPZACXCFCXCUGAJUHUIUJUKAWTRZFXCCXEDULPZFBDUMQZGUBZUNZUOPZFOUPZ
+      UAUPZDUQQZURZDUSQZUTZFXKXOUTZFXLXOUTZVAZVBZUABVCOBVCZXDAXFWTLSZXEFBXIAFBP
+      ZWTMSZXEFXGPZVDZFXHPVDZFXIPVDZXEYEWSBUGZXEYIYEXEBDXGWSEFXGTZKWSTIYDYBVEVF
+      XEYERWSBXEWSBVGZYEAWTDVMPZYKXEDYBVHZBWSDXMIXMTZVIVJSVKVLAYGWTAFGVGYGNFGVN
+      VOSXIXIUGYHYFYGRWDXITXIXGXHFVPVQVRVSXEXTOUABBXEXKBPZXLBPZXTXEYORZYPRZXPXS
+      YRXPRZXKWSPZXLWSPZXQXRYSYTXQYSBXODEFXKIKXOTZXEYLYOYPXPYMVTZAYCWTYOYPXPMWA
+      ZWBUJYSUUAXRYSBXODEFXLIKUUBUUCUUDWBUJYSXAWTYOYPXNWSPZYTUUAVAAXAWTYOYPXPXB
+      WAAWTYOYPXPWCXEYOYPXPWEYQYPXPWFYRUUEXPYRBXODEFXNIKUUBXEYLYOYPYMWGAYCWTYOY
+      PMVTWBVFBWSDXMXKXLIYNWHWIWJWKWLWMXFXDXJYAROUABXOFDXMXGULGIYJHUUBYNWNVFWOJ
+      WPWQ $.
+  $}
+
+  ${
+    rprmndvdsr1.1 $e |- .1. = ( 1r ` R ) $.
+    rprmndvdsr1.2 $e |- .|| = ( ||r ` R ) $.
+    rprmndvdsr1.3 $e |- P = ( RPrime ` R ) $.
+    rprmndvdsr1.4 $e |- ( ph -> R e. CRing ) $.
+    rprmndvdsr1.5 $e |- ( ph -> Q e. P ) $.
+    $( A ring prime element does not divide the ring multiplicative unit.
```

Comment:
"ring unity" or "ring multiplicative identity"

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4837/commits/3cb77ca77c03e28c1c877ddab4daed48dd91fb30